package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Amazon Echo skill for the Washington D.C. Metro",
   "main": "index.js",
   "scripts": {
-    "test": "tape test/*.js | faucet"
+    "test": "tape test/*.js | faucet",
+    "test:record": "NOCK_BACK_MODE=record npm test",
```

Comment:
Running `npm run test:record` will make an actual http request and save the data in the `test/fixtures` directory. Then when you run `npm test` or `gulp test`, it will use that data so that real requests are not made.